From a58354b7fedc44473306ffd5ac9af00936fda97a Mon Sep 17 00:00:00 2001
From: Wladimir Leuschner <wladimir.leuschner@qt.io>
Date: Wed, 3 Sep 2025 12:27:06 +0200
Subject: [PATCH 214/553] WindowsQPA: Remove unused GDIPlus include from
 qwindowswindow.cpp

GDIPlus is not used anymore to draw the custom titlebar, therefore the
gdiplus include can be removed.

Fixes: QTBUG-139653
Pick-to: 6.9
Change-Id: Id80b9d58bd6729dad8a4a428333e41d41aec5cd0
Reviewed-by: Joerg Bornemann <joerg.bornemann@qt.io>
(cherry picked from commit e6f8cc8598bcce571891d90dfd1dde7ccb20e7ea)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/plugins/platforms/windows/qwindowswindow.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/plugins/platforms/windows/qwindowswindow.cpp b/src/plugins/platforms/windows/qwindowswindow.cpp
index b94c809eda2..12f34eec265 100644
--- a/src/plugins/platforms/windows/qwindowswindow.cpp
+++ b/src/plugins/platforms/windows/qwindowswindow.cpp
@@ -44,7 +44,6 @@
 #include <QtCore/qoperatingsystemversion.h>
 
 #include <dwmapi.h>
-#include <gdiplus.h>
 
 #if QT_CONFIG(vulkan)
 #include "qwindowsvulkaninstance.h"
-- 
2.51.2

